application/world/services/auto_bible_generator.py

```
@@ -178,24 +188,11 @@ async def generate_and_save(
         logger.info(f"Generating Bible for novel: {premise[:50]}... (stage: {stage})")
 
         # 1. 创建空 Bible（如果不存在）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)